package.json

```diff
@@ -5,20 +5,21 @@
   "main": "server.js",
   "scripts": {
     "clean": "rimraf dist",
+    "css": " sass src/public/scss/app.scss src/public/css/app.css",
     "copy-assets": "ts-node tools/copy-assets",
     "tsc": "tsc",
     "prettier:ci": "prettier --config .prettierrc src/**/*.ts --check",
     "prettier:fix": "prettier --config .prettierrc src/**/*.ts --write",
     "lint:ci": "eslint .",
     "lint:fix": "eslint . --fix",
-    "build": "npm-run-all clean tsc copy-assets",
+    "build": "npm-run-all clean tsc css copy-assets",
```

Comment:
Styles are now in `/src/publc/scss/app.scss` and get automatically compiled to css on save when `npm run dev` is active, or on `npm run build` as part of the deploy.